js/build_map.js

```diff
@@ -134,32 +163,70 @@ function fillMap() {
-    
-  buttonContainer.append('div')
-    .classed('select-arrowbox', true);
+function resize() {
+  
+  // Decide whether we're in mobile or desktop mode. Currently doing this by window width, but we could look to
+  // https://www.w3schools.com/howto/howto_js_media_queries.asp for more device-specific solutions
+  if(window.innerWidth > 425) { // sufficiently wide desktop windows
+    waterUseViz.mode = 'desktop';
+  } else { // most mobile devices (except iPads) plus narrow desktop windows
+    waterUseViz.mode = 'mobile';
+  }
+  
+  // Calculate new dimensions with adaptations for ~desktop vs ~mobile
+  if(waterUseViz.mode === 'desktop') {
+  
+    // buttonBox is at the left and centered vertically
+    waterUseViz.dims.buttonBox.y0 = (waterUseViz.dims.map.height/2) - (waterUseViz.dims.buttonBox.height/2);
+    waterUseViz.dims.buttonBox.width = waterUseViz.dims.buttonBox.widthDesktop;
+    waterUseViz.dims.buttonBox.height = waterUseViz.dims.buttonBox.heightDesktop;
+    // map fills the full svg
+    waterUseViz.dims.map.x0 = waterUseViz.dims.buttonBox.width;
+    // svg is [buttons][map]
+    waterUseViz.dims.svg.width = waterUseViz.dims.buttonBox.width + waterUseViz.dims.map.width;
+    waterUseViz.dims.svg.height = waterUseViz.dims.map.height;
+    // watermark is at bottom left
+    waterUseViz.dims.watermark.x0 = waterUseViz.dims.svg.width * 0.01;
+    waterUseViz.dims.watermark.y0 = waterUseViz.dims.svg.height * 0.95;
     
-  d3.select('#button-container')
-    .selectAll('button')
-    .data(categories)
-    .enter()
-    .append('button')
-    .text(function(d){
-      return categoryToName(d);
-    })
-    .attr('class', function(d){
-      return d;
-    })
-    .on('click', function(d){
-      updateCategory(d.toLowerCase(), activeCategory);
-    })
-    .on('mouseover', function(d){
-      showCategory(d.toLowerCase(), activeCategory, action = 'mouseover');
-    })
-    .on('mouseout', function(d){
-      showCategory(activeCategory, d.toLowerCase(), action = 'mouseout');
-    });
+  } else {
+  
+    // buttonBox sits below map with small vertical buffer between map and buttons
+    waterUseViz.dims.buttonBox.y0 = waterUseViz.dims.map.height * 1.05;
+    waterUseViz.dims.buttonBox.width = waterUseViz.dims.map.width;
+    waterUseViz.dims.buttonBox.height = waterUseViz.dims.buttonBox.width *
+      (waterUseViz.dims.buttonBox.heightDesktop / waterUseViz.dims.buttonBox.widthDesktop);
+    // map fills the top part of the svg
+    waterUseViz.dims.map.x0 = 0;
+    // svg is [map]
+    //        [buttons]
+    waterUseViz.dims.svg.width = waterUseViz.dims.map.width;
+    waterUseViz.dims.svg.height = waterUseViz.dims.buttonBox.y0 + waterUseViz.dims.buttonBox.height;
+    // watermark is at bottom right
+    waterUseViz.dims.watermark.x0 = waterUseViz.dims.svg.width * 0.85;
+    waterUseViz.dims.watermark.y0 = waterUseViz.dims.svg.height * 0.95;
+  }
+  
+  // Apply the changes to the svg, map, map background, and watermark
```

Comment:
So are these things global vars? Should they either be in `waterUseViz` or passed into this function?

Reply:
they are globals, and i'd like to get them in waterUseViz eventually, but I'm trying not to change too too much code in this PR, and #127 is a separate (backlogged) issue. so i'm really just trying to lay the groundwork so that someone can complete the transition to using `waterUseViz` in the future.